### PR TITLE
add First conference for scientific programming

### LINKED ISSUE #313: Add sciwork 2023 conference report
Please add the sciwork 2023 conference report (https://docs.google.com/document/d/1iKvwd9VLiyqjUaeJzk7GkqyKwbC8oCk1qSVrPvjEw1M) to the portal site.  It can be made available under the categories https://sciwork.dev/conference/ .

### PR DESCRIPTION
**Pull request by tigercosmos**
fix #313

### CODE REVIEW

**Comment by tigercosmos** [comment]
@yungyuc please review.

File: content/pages/conference/report.rst
Comment (yungyuc) [inline content/pages/conference/report.rst:1]: Please add year in the path, e.g., `content/pages/conference/2023/report.rst`.

File: content/pages/conference/report.rst [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+The community of sciwork focuses on scientific programming, which is a less visible area. 
+Electronic computers were invented for engineering purposes,
+and programming for science and engineering is not only important for solving the problems,
+but also leads to advances to computing itself.
+Although the rewards are huge, it does take a lot of effort to get into the business of scientific software development.
+
+
+
+.. figure:: https://github.com/sciwork/swportal/assets/18013815/c55d5183-0989-4521-a013-6115c3278800
```
Comment (yungyuc) [inline content/pages/conference/report.rst:86]: Assets should be added to the repository along with the text.

File: content/pages/conference/report.rst [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+
+Event Album
+----------------------------------------
+
+If you missed it, check the photos at https://www.flickr.com/photos/sciwork/albums.
+
+
+We will also upload the recordings once they are processed.
+
+.. raw:: html
```
Comment (yungyuc) [inline content/pages/conference/report.rst:30]: The embedded HTML looks a little bit strange on mobile.  It should be OK at the moment.  But it is also good to improve.

**Review by yungyuc — CHANGES_REQUESTED** [review]
Thanks, @tigercosmos .  Points to be addressed:

- [ ] Add year in report path.
- [ ] Add assets to the repository.

File: content/pages/conference/report.rst (hunk 2 above)
Comment (tigercosmos) [reply content/pages/conference/report.rst]: image

image

Comment (tigercosmos) [reply content/pages/conference/report.rst]: fixed.

File: content/pages/conference/report.rst (hunk 1 above)
Comment (tigercosmos) [reply content/pages/conference/report.rst]: fixed.

File: content/pages/conference/report.rst
Comment (tigercosmos) [reply content/pages/conference/report.rst:1]: fixed.

File: content/pages/conference/report.rst (hunk 2 above)
Comment (yungyuc) [reply content/pages/conference/report.rst]: Your skill works like a magic!  Impressed.

File: content/pages/conference/2023/report.rst [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+
+The community of sciwork focuses on scientific programming, which is a less visible area. 
+Electronic computers were invented for engineering purposes,
+and programming for science and engineering is not only important for solving the problems,
+but also leads to advances to computing itself.
+Although the rewards are huge, it does take a lot of effort to get into the business of scientific software development.
+
+
+
+.. figure:: {attach}talk-in-a-snapshot.png
```
Comment (yungyuc) [inline content/pages/conference/2023/report.rst:92]: The figure should be centered.  But this is minor.  Let's leave it here.

**Review by yungyuc — APPROVED** [review]
Looks great.  Thanks, @tigercosmos

**Pull request merged**